src/components/directory-content.js

```diff
@@ -673,6 +681,45 @@ const DirectoryContent = () => {
         );
     };
 
+    const sort = useCallback(
+        (datakey, isAsc, isNumeric) => {
+            const rowsCopy = [...rows].map((row, index) => [row, index]);
```

Comment:
```suggestion
            const rowsCopy = rows.map((row, index) => [row, index]);
```
I think it's ok like that no ?